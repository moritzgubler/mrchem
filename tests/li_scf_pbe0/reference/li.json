{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            0.0
          ],
          "radius": 2.05
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "li",
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        }
      ],
      "multiplicity": 2
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 5,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -4
    },
    "printer": {
      "file_name": "li.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (PBE0)",
        "prec": 0.001,
        "restricted": false,
        "screen": 12.0,
        "type": "gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": -1.0,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 0.001,
        "helmholtz_prec": -1.0,
        "kain": 3,
        "light_speed": -1.0,
        "localize": false,
        "max_iter": 5,
        "method": "DFT (PBE0)",
        "orbital_thrs": 0.02,
        "proj_prec": 0.001,
        "rotation": 0,
        "shared_memory": false,
        "smooth_prec": 0.001,
        "start_prec": 0.001
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        0.0
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 7.580206233951303e-14,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            0.0,
            0.0,
            0.0
          ],
          "vector_el": [
            0.0,
            0.0,
            0.0
          ],
          "vector_nuc": [
            0.0,
            0.0,
            0.0
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Li",
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        }
      ],
      "multiplicity": 2,
      "orbital_energies": {
        "energy": [
          -2.0559145926589957,
          -0.14107124869558188,
          -2.0456268559566233
        ],
        "occupation": [
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "b"
        ],
        "sum_occupied": -4.242612697311201
      },
      "scf_energy": {
        "E_ee": 4.048665717410053,
        "E_eext": 0.0,
        "E_el": -7.466159701873801,
        "E_en": -17.088404230890806,
        "E_kin": 7.379879336043726,
        "E_next": 0.0,
        "E_nn": 0.0,
        "E_nuc": 0.0,
        "E_tot": -7.466159701873801,
        "E_x": -0.4427266703348705,
        "E_xc": -1.363573854101903,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 1,
      "routine": "mrchem.x",
      "total_cores": 1,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 4.061916632919257,
        "E_eext": 0.0,
        "E_el": -7.413768850996542,
        "E_en": -17.035029696824992,
        "E_kin": 7.3709974445524,
        "E_next": 0.0,
        "E_nn": 0.0,
        "E_nuc": 0.0,
        "E_tot": -7.413768850996542,
        "E_x": -0.4447685124391083,
        "E_xc": -1.3668847192040994,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 4.035331948676242,
              "E_eext": 0.0,
              "E_el": -7.465238166134573,
              "E_en": -17.01159653990369,
              "E_kin": 7.311304561296245,
              "E_next": 0.0,
              "E_nn": 0.0,
              "E_nuc": 0.0,
              "E_tot": -7.465238166134573,
              "E_x": -0.4413848202394499,
              "E_xc": -1.3588933159639214,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -7.465238166134573,
            "energy_update": 0.05146931513803121,
            "mo_residual": 0.06988028196837986,
            "wall_time": 16.92174498
          },
          {
            "energy_terms": {
              "E_ee": 4.048665717410053,
              "E_eext": 0.0,
              "E_el": -7.466159701873801,
              "E_en": -17.088404230890806,
              "E_kin": 7.379879336043726,
              "E_next": 0.0,
              "E_nn": 0.0,
              "E_nuc": 0.0,
              "E_tot": -7.466159701873801,
              "E_x": -0.4427266703348705,
              "E_xc": -1.363573854101903,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -7.466159701873801,
            "energy_update": 0.000921535739228041,
            "mo_residual": 0.017571466544120923,
            "wall_time": 11.975392115
          }
        ],
        "wall_time": 28.897376347
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
