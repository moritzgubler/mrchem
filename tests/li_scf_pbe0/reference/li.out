                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         rebase-zora                              ***
***         Git commit hash    faecb83984c1d010ec90-dirty               ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Thu Mar 10 17:22:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               1
 Total cores             :                                               1
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (1 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project GTO molecular orbitals
 Precision               : 1.00000e-03
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 MO alpha file           : initial_guess/mrchem.moa
 MO beta file            : initial_guess/mrchem.mob
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               2
 Beta electrons          :                                               1
 Total electrons         :                                               3
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999000474e-01
    1    1    a          :                              1.000000190643e+00
    2    1    b          :                              9.999998992098e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  7.370997444552
 E-N energy              :            (au)                -17.035029696825
 Coulomb energy          :            (au)                  4.061916632919
 Exchange energy         :            (au)                 -0.444768512439
 X-C energy              :            (au)                 -1.366884719204
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  0.000000000000
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -7.413768850997
 Nuclear energy          :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.413768850997e+00
                         :      (kcal/mol)             -4.652210154978e+03
                         :        (kJ/mol)             -1.946484730106e+04
                         :            (eV)             -2.017389188762e+02
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                 -2.026209362322
    1    1    a          :            (au)                 -0.139783728888
    2    1    b          :            (au)                 -2.015925490764
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -4.181918581974
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Checkpointing           : Off
 Max iterations          : 5
 KAIN solver             : 3
 Localization            : Off
 Diagonalization         : First two iterations
 Start precision         : 1.00000e-03
 Final precision         : 1.00000e-03
 Helmholtz precision     : Dynamic
 Energy threshold        : Off
 Orbital threshold       : 2.00000e-02
 ZORA potential          : None
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          1.732051e+00          -7.413768850997         -7.413769e+00
    1          6.988028e-02          -7.465238166135         -5.146932e-02
    2          1.757147e-02          -7.466159701874         -9.215357e-04
---------------------------------------------------------------------------
                      SCF converged in 2 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  7.379879336044
 E-N energy              :            (au)                -17.088404230891
 Coulomb energy          :            (au)                  4.048665717410
 Exchange energy         :            (au)                 -0.442726670335
 X-C energy              :            (au)                 -1.363573854102
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  0.000000000000
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -7.466159701874
 Nuclear energy          :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.466159701874e+00
                         :      (kcal/mol)             -4.685085909992e+03
                         :        (kJ/mol)             -1.960239946013e+04
                         :            (eV)             -2.031645464925e+02
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                 -2.055914592659
    1    1    a          :            (au)                 -0.141071248696
    2    1    b          :            (au)                 -2.045626855957
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -4.242612697311
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m 36s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
